camunda.Dockerfile

```
@@ -0,0 +1,165 @@
+     --strip-debug \
+     --no-man-pages \
+     --no-header-files \
+     --compress=2 \
+     --output /jre && \
+   rm -rf /jre/lib/src.zip
+
+### Java base image
+FROM base AS java
+WORKDIR /
+
+# Inherit from previous build stage
+ARG JAVA_HOME=/opt/java/openjdk
+
+# Default to UTF-8 file encoding
+ENV LANG='C.UTF-8' LC_ALL='C.UTF-8'
+
+# Setup JAVA_HOME and binaries in the path
+ENV JAVA_HOME ${JAVA_HOME}
+ENV PATH $JAVA_HOME/bin:$PATH
+
+# Copy JRE from previous build stage
+COPY --from=jre-build /jre ${JAVA_HOME}
+
+# https://github.com/docker-library/openjdk/issues/212#issuecomment-420979840
+# https://openjdk.java.net/jeps/341
+# TL;DR generate some class data sharing for faster load time
+RUN java -Xshare:dump;
+
+### Build camunda from scratch ###
+FROM java as build
+WORKDIR /camunda
+ENV MAVEN_OPTS -XX:MaxRAMPercentage=80
+COPY --link . ./
+RUN --mount=type=cache,target=/root/.m2,rw \
+    ./mvnw -B -am -pl dist package -T1C -D skipChecks -D skipTests && \
+    mv dist/target/camunda-zeebe .
+
+### Extract camunda from distball ###
+# hadolint ignore=DL3006
+FROM base as distball
+WORKDIR /camunda
+ARG DISTBALL="dist/target/camunda-zeebe-*.tar.gz"
+COPY --link ${DISTBALL} camunda.tar.gz
+
+# Remove zbctl from the distribution to reduce CVE related maintenance effort w.r.t to containers
+RUN mkdir camunda-zeebe && \
+    tar xfvz camunda.tar.gz --strip 1 -C camunda-zeebe && \
+    find . -type f -name 'zbctl*' -delete
+
+### Image containing the camunda distribution ###
+# hadolint ignore=DL3006
+FROM ${DIST} as dist
+
+### Application Image ###
+# https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
+# hadolint ignore=DL3006
+FROM java as app
+# leave unset to use the default value at the top of the file
+ARG BASE_IMAGE
+ARG BASE_DIGEST
+ARG VERSION=""
+ARG DATE=""
+ARG REVISION=""
+
+# OCI labels: https://github.com/opencontainers/image-spec/blob/main/annotations.md
+LABEL org.opencontainers.image.base.digest="${BASE_DIGEST}"
+LABEL org.opencontainers.image.base.name="docker.io/library/${BASE_IMAGE}"
+LABEL org.opencontainers.image.created="${DATE}"
+LABEL org.opencontainers.image.authors="zeebe@camunda.com"
+LABEL org.opencontainers.image.url="https://zeebe.io"
+LABEL org.opencontainers.image.documentation="https://docs.camunda.io/docs/self-managed/zeebe-deployment/"
+LABEL org.opencontainers.image.source="https://github.com/camunda/camunda"
+LABEL org.opencontainers.image.version="${VERSION}"
+# According to https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys
+# and given we set the base.name and base.digest, we reference the manifest of the base image here
+LABEL org.opencontainers.image.ref.name="${BASE_IMAGE}"
+LABEL org.opencontainers.image.revision="${REVISION}"
+LABEL org.opencontainers.image.vendor="Camunda Services GmbH"
+LABEL org.opencontainers.image.licenses="(Apache-2.0 AND LicenseRef-Zeebe-Community-1.1)"
```

The license reference must point to the Camunda License 1.0

```suggestion
LABEL org.opencontainers.image.licenses="(Apache-2.0 AND LicenseRef-Camunda-License-1.0)"
```

(Just noticed, the same must be done in the already existing Dockerfiles)

---

Thanks I will update the other dockerfiles as well

---

Done